### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by lekegitrepo**
### Milestone 2: User interface

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  p "Hello #{p1name}, you will play with X"
+
+  p 'Player 2 please enter your name: '
+  p2name = gets.chomp
+  p "Hello #{p2name}, you will play with O"
+
+  grid = Board.new
+  grid.display
+
+  p 'Player 1 Turn, please select your next move: '
```
Comment (samgaco) [inline bin/main.rb]: * [**REQUIRED**] game asks player to select from available moves

what moves do we have available? the board doesn't show any reference for that, should they be numbers from 1 to 9? letters?

File: bin/main.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  p1turn = gets.chomp.to_i
```
Comment (samgaco) [inline bin/main.rb]: * [**REQUIRED**] game informs player if selected move is invalid

what if I input: "**ojfiwfjwef**" , the game should not accept that input for a turn

File: bin/main.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+  grid.display
+
+  p 'Player 2 Turn, please select your next move: '
+  p2turn = gets.chomp.to_i
+  grid.insertMove(p2turn, 'O')
+
+  grid.display
+
+  ending = p 'Thank you for playing Tic Tac Toe'
```
Comment (samgaco) [inline bin/main.rb]: [**REQUIRED**] The game should pic a winner or check if there was a draw 


[**SUGGESTION**] of course is mocked, maybe the number 7 could be a win, and picking all the numbers except 7 could be a draw

File: bin/main.rb (hunk 1 above)
Comment (samgaco) [inline bin/main.rb]: * [**REQUIRED**] game repeats all actions for next player's move

Think about how will you achieve the mockup to keep iterating until it finds a winner or the game is finally set as a draw. **hint**: a loop :D

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi @lekegitrepo and @noomdalv :high_brightness: 

I Left some comments below and requested changes, all the comments are required, the comment tagged as a suggestion is just my own opinion and you could implement this requirement using any other approach.

**Things to work on:**

* Have into account, that each file should consist of only one class.
* game asks player to select from available moves
* game informs player if selected move is invalid
* The game should pic a winner or check if there was a draw
* game repeats all actions for next player's move

Please :pray: work on those and submit back for another code review once you are done.
-----------------
moving on :arrow_down: 
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description


------------------

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Review by Ispirett — APPROVED** [review]
Hey, @noomdalv @lekegitrepo really good job you can move on to the next milestone. :100:

**Pull request merged**